docs/AndroidBuildingFromSource.md

```diff
@@ -20,13 +20,25 @@ Make sure you have the following installed:
 3. Local Maven repository for Support Libraries (formerly `Android Support Repository`) >= 17 (for Android Support Library)
 4. Android NDK (download links and installation instructions below)
 
-Point Gradle to your Android SDK: either have `$ANDROID_SDK` and `$ANDROID_NDK ` defined, or create a local.properties file in the root of your react-native checkout with the following contents:
+### Point Gradle to your Android SDK:
+
+**Step 1:**  Set environment variables through your local shell.
```

Comment:
Yea I originally split it up into 2 options but took @satya164 original suggestions that is required in most cases with android studio and remove my original note.